.storybook/stories/section-title.stories.mdx

```diff
@@ -0,0 +1,68 @@
+import { Meta, Story, Canvas, ArgsTable} from '@storybook/addon-docs/blocks';
+
+import { withKnobs, boolean, object, } from "@storybook/addon-knobs";
+import { withA11y } from '@storybook/addon-a11y';
```

Comment:
We can remove `withA11y`, `withKnobs`, and `withActions`  imports as of the Storybook 6 upgrade